lambeq/text2diagram/ccg_types.py

```diff
@@ -77,15 +77,15 @@ def str2biclosed(cat: str, str2type: Callable[[str], Ty] = Ty) -> Ty:
     ----------
     cat : str
         The string to be parsed.
-    str2type: callable, default: discopy.grammar.categorial.Ty
-        A function that parses an atomic category into a categorial type.
-        The default uses :py:class:`discopy.grammar.categorial.Ty` to
+    str2type: callable, default: discopy.biclosed.Ty
+        A function that parses an atomic category into a biclosed type.
+        The default uses :py:class:`discopy.biclosed.Ty` to
         produce a type with the same name as the atomic category.
 
     Returns
     -------
-    discopy.grammar.categorial.Ty
-        The parsed category as a categorial type.
+    discopy.biclosed.Ty
```

Comment:
DisCoPy v1.0 renamed `biclosed` to `closed`, which is part of the reason why I chose to rename them to `categorial` (which is also coherent with renaming `rigid.Ty` to `pregroup.Ty`)

Reply:
For context, the motivation for renaming `biclosed` to `closed` was that it's implicit that things are closed on both sides, the same way that it's implicit that things are rigid on both sides, and nobody wants to call a pregroup a `birigid` category.

Reply:
I agree on principle, this is just going to be a notable change in the interface, so I'd like to deal with it in its own PR